lang/icon/Portfile

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
-name                icon
-version             9.5.1
+github.setup        gtownsend icon 9.5.24b v
```

Comment:
this should be:
```
github.setup        gtownsend icon 9.5.24 v b
```
and then the lines below for `master_sites` and `distname` need to be removed

Reply:
> `github.setup      gtownsend icon 9.5.24 v b`

> and then the lines below for master_sites and distname need to be removed

Done.  Thanks for advice, I had a lot of trouble figuring this out because of GitHub magic renaming of the fetch URL.